src/logmessage.cpp

```diff
@@ -136,14 +140,15 @@ namespace g3 {
 
    LogMessage::LogMessage(const std::string& file, const int line,
                           const std::string& function, const LEVELS& level)
-      : _timestamp(g3::systemtime_now())
-      , _call_thread_id(std::this_thread::get_id())
+      : _call_thread_id(std::this_thread::get_id())
       , _microseconds(microsecondsCounter())
```

Comment:
I agree, microsec counter is useless after this request is complete. I will remove it.